pkcs11/util_pkcs11.c

```diff
@@ -2147,14 +2150,9 @@ CK_RV apply_sign_mechanism_finalize(yubihsm_pkcs11_op_info *op_info) {
   return CKR_OK;
 }
 
-CK_RV apply_verify_mechanism_finalize(yubihsm_pkcs11_op_info *op_info
-                                      __attribute((unused))) {
+CK_RV apply_verify_mechanism_finalize() { return CKR_OK; }
```

Comment:
Where did the argument go here?

Reply:
I put it back and used the UNUSED macro.

(MSVC did not like __attribute((unused)) and gnu didn't like an unused argument that wasn't marked as unused)